rfcs/0166-nix-formatting.md

```diff
@@ -262,34 +262,34 @@ Terms and definitions:
 - Braces: `{}`
 - Parentheses: `()`
 
-- Line breaks may not be preserved, but empty lines must always be preserved.
-  - This allows the formatter to compact down multi-line expressions if necessary, while still allowing to structure the code appropriately.
-  - Multiple subsequent empty lines must get collapsed into a single one.
+- Line breaks may be added or removed. Consecutive empty lines must be turned into a single empty line.
```

Comment:
Sick of me yet? :sweat_smile: Consecutive empty lines implies more than one, I think, so this seems to allow a formatter to remove a single empty line if it chooses to, or insert new single empty lines, which I suspect is not the intent.

```suggestion
- Line breaks may be added or removed, but empty lines must not be created. Single empty lines must be preserved, and consecutive empty lines must be turned into a single empty line.
```

Reply:
Sounds good to me!